src/python/botan.py

```diff
@@ -15,7 +16,7 @@
 """
 Module initialization
 """
-botan = CDLL('libbotan-1.11.so')
+botan = CDLL('libbotan-1.11.dylib')
 
```

Comment:
I think this should be somehow platform independent. Does that work in Windows as well?